src/main/java/gregtech/common/metatileentities/multi/electric/generator/MetaTileEntityLargeCombustionEngine.java

```diff
@@ -80,11 +75,11 @@ protected BlockPattern createStructurePattern() {
                 .aisle("XCX", "CGC", "XCX")
                 .aisle("XCX", "CGC", "XCX")
                 .aisle("AAA", "AYA", "AAA")
```

Comment:
This structure pattern does not account for the muffler at all.

Reply:
fixed